db.js

```diff
@@ -255,20 +262,23 @@ export class ActivityPubDB extends EventTarget {
     const store = tx.objectStore(NOTES_STORE)
     const totalNotes = await store.count()
 
```

Comment:
I think we can get rid of the uniqueIndexes set and just allow duplicates for now.

Reply:
But that will be a bad UX :/

Reply:
Yeah, lets add a todo for later and see how it goes? I think performance and maintenance should come first. I worry our algorithm can get stuck in a loop trying to fill slots at random without duplicates as the amount of data grows. I think "cpu heats up as the timeline is stuck loading" is a likely outcome with how the code was.